### PR TITLE
Reorganization

### PR DESCRIPTION
**Pull request by lucalavezzo**
See #283

### CODE REVIEW

File: additional_tools/unit_tests/test_ntuplemaker.py [hunk 1]
```diff
@@ -33,11 +33,11 @@
         "singularity": "/cvmfs/unpacked.cern.ch/registry.hub.docker.com/coffeateam/coffea-dask:latest",
         "script": "condor_SUEP_WS.py",
         "options": [
-            "--isMC 1 --era 2018 --doSyst 1",
+            "--isMC 1 --era 2016 --doSyst 1",
             "--isMC 0 --era 2018 --doSyst 1",
```
Comment (chadfreer) [inline additional_tools/unit_tests/test_ntuplemaker.py:37]: Actually what is the isMC 0 doSyst 1 doing?

File: histmaker/make_hists.py [hunk 2]
```diff
@@ -14,20 +14,24 @@
...
+import sys
 
-import fill_utils
-import hist_defs
 import numpy as np
-import plot_utils
 import uproot
+from tqdm import tqdm
+
+sys.path.append("..")
```
Comment (chadfreer) [inline histmaker/make_hists.py:23]: Why is this necessary?

File: histmaker/submit.py [hunk 3]
```diff
@@ -21,11 +21,15 @@
 import os
 import shlex
 import subprocess
+import sys
 from multiprocessing.pool import Pool, ThreadPool
 
 import numpy as np
+
+sys.path.append("..")
```
Comment (chadfreer) [inline histmaker/submit.py:29]: Same question here

File: plotting/plot_utils.py [hunk 4]
```diff
@@ -17,6 +17,9 @@
 import uproot
 from sympy import diff, sqrt, symbols
 
+sys.path.append("..")
```
Comment (chadfreer) [inline plotting/plot_utils.py:20]: same here

File: workflows/SUEP_coffea.py [hunk 5]
```diff
@@ -30,6 +28,9 @@
     track_killing,
 )
 
+# IO utils
+from workflows.utils import pandas_utils
+
```
Comment (chadfreer) [inline workflows/SUEP_coffea.py:33]: Why is this moved later? Also this change needs to be added the the other workflows.

File: additional_tools/unit_tests/test_ntuplemaker.py (hunk 1 above)
Comment (lucalavezzo) [reply additional_tools/unit_tests/test_ntuplemaker.py:37]: Maybe this script could be made a little simpler, but the idea is that for this analysis, we run two commands, and for this test we thus define two input files, and two sets of options to pass to the script: one for 2016 and one for 2018 (and both running systematics).

File: histmaker/make_hists.py (hunk 2 above)
Comment (lucalavezzo) [reply histmaker/make_hists.py:23]: It's for `import plotting.plot_utils`.
